.github/workflows/Documentation.yaml

```diff
@@ -128,7 +131,5 @@ jobs:
           YQ_BINARY: yq_linux_amd64
         run: |
           set -ex
-          wget https://github.com/mikefarah/yq/releases/download/${{ env.YQ_VERSION }}/${{ env.YQ_BINARY }}
-          sudo mv ${{ env.YQ_BINARY }} /usr/bin/yq
-          sudo chmod +x /usr/bin/yq
-          ./src/docs/utils/publish_docs.sh ${{ needs.validate-documentation-request.outputs.oci-img-name }} 
+          ./src/notifications/requirements.sh
+          ./src/docs/publish_docs.sh ${{ needs.validate-documentation-request.outputs.oci-img-name }} ${{ needs.validate-documentation-request.outputs.oci-img-name }}_doc_data data
```

Comment:
You've just moved `_doc_data` and `data` from inside the script to the workflow, but they are still in plain text.

If tomorrow we change `https://github.com/canonical/oci-factory/blob/df8f40f9f65fdb7c122c1ed8d76fab7b6bedec77/src/docs/generate_oci_doc_yaml.py#L332` this will all fail. It needs to be a variable.